packages/utils/interactive-hooks/package.json

```diff
@@ -24,17 +24,15 @@
     "invariant": "^2.2.0"
   },
   "devDependencies": {
-    "@office-iss/react-native-win32": "0.62.6",
     "@types/react": "^16.9.34",
```

Comment:
Don't forget to upgrade typings for react + react-native as well.

Reply:
Right. Those versions should just correspond with the same version of react/react native yes?